datasync/chaindatafetcher/kas/repository_contracts.go

```diff
@@ -36,24 +36,23 @@ func filterKIPContracts(api BlockchainAPI, event blockchain.ChainEvent) ([]*FT,
 		if receipt.ContractAddress == (common.Address{}) {
 			continue
 		}
-		blockNumber := event.Block.Number()
 		contract := receipt.ContractAddress
-		isKIP13, err := caller.isKIP13(contract, blockNumber)
+		isKIP13, err := caller.isKIP13(contract, nil)
```

Comment:
```suggestion
		isKIP13, err := caller.isKIP13(contract, rpc.LatestBlockNumber)
```

Reply:
The type of parameter is *big.Int and it will convert to rpc.BlockNumber in CodeAt and CallContract.